src/playground/components/Configuration.js

```diff
@@ -105,10 +105,35 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
         value: ruleName,
         label: rulesMeta[ruleName].deprecated ? ruleName.concat(" (deprecated)") : ruleName
     }));
-    const [selectedRule, setSelectedRule] = useState();
+    const [selectedRules, setSelectedRules] = useState([]);
     const ruleInputRef = useRef(null);
     const [rulesWithInvalidConfigs, setRulesWithInvalidConfigs] = useState(new Set([]));
 
+    const handleRuleChange = () => {
+        selectedRules.forEach(selectedRule => {
+            if (ruleNames.includes(selectedRule)) {
+                options.rules[selectedRule] = ["error"];
+                onUpdate(Object.assign({}, options));
+                ruleInputRef.current.setValue("");
+            }
+        });
+    };
+
+    const Input = props => {
+        if (props.isHidden) {
+            return <components.Input {...props} />;
+        }
+        return (
+            <components.Input {...props} onKeyDown={e => {
+                if ((e.metaKey || e.ctrlKey) && e.code === "Enter") {
```

Comment:
when `Enter` from numpad is pressed, `e.code` will be `"NumpadEnter"`
```suggestion
                if ((e.metaKey || e.ctrlKey) && e.key === "Enter") {
```